Resources/public/js/views/ez-dashboardblockbasedview.js

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockbasedview', function (Y) {
+    'use strict';
+    /**
+     * Provides the Dashboard Block Based View class
+     *
+     * @module ez-dashboardblockbasedview
+     */
+    Y.namespace('eZ');
+
+    var SELECTOR_CONTENT = '.dashboard-content';
+
+    /**
+     * The dashboard view
+     *
+     * @namespace eZ
+     * @class DashboardBlockBasedView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.DashboardBlockBasedView = Y.Base.create('dashboardBlockBasedView', Y.eZ.TemplateBasedView, [], {
+        initializer: function () {
+            this.after('activeChange', function () {
+                this.after('blocksChange', this._renderBlocks, this);
+
+                this._makeBlocksActive();
+            }, this);
+        },
+
+        /**
+         * Renders the dashboard view
+         *
+         * @method render
+         * @return {eZ.DashboardBlockBasedView} the view itself
+         */
+        render: function () {
+            this.get('container').setHTML(this.template({dashboardTitle: this.get('dashboardTitle')}));
```

Comment:
Why? If you want to translate the title, then you have to store it in some kind of variable. If you hardcode the `Dashboard` text inside the template, then it will be impossible.